src/Network/Receive/kRO/RagexeRE_2020_03_04a.pm

```diff
@@ -0,0 +1,65 @@
+#
+#  This software is open source, licensed under the GNU General Public
+#  License, version 2.
+#  Basically, this means that you're allowed to modify and distribute
+#  this software. However, if you distribute modified versions, you MUST
+#  also distribute the source code.
+#  See http://www.gnu.org/licenses/gpl.html for the full license.
+########################################################################
+# Korea (kRO) #bysctnightcore
+# The majority of private servers use eAthena, this is a clone of kRO
+package Network::Receive::kRO::RagexeRE_2020_03_04a;
+use strict;
+use base qw(Network::Receive::kRO::Ragexe_2018_11_14c);
+
+sub new {
+	my ($class) = @_;
+	my $self = $class->SUPER::new(@_);
+    # todo
+    my %packets = (
+		'0097' => ['private_message', 'v Z24 V Z*', [qw(len privMsgUser flag privMsg)]], # -1
+		
+		'009D' => ['item_exists', 'a4 V C v3 C2', [qw(ID nameID identified x y amount subx suby)]], # 17
+		'0ADD' => ['item_appeared', 'a4 V v C v2 C2 v C v', [qw(ID nameID type identified x y subx suby amount show_effect effect_type )]],
+
+
+		'01C8' => ['item_used', 'a2 V a4 v C', [qw(ID itemID actorID remaining success)]], # 13
+
+
+		'09FD' => ['actor_moved', 'v C a4 a4 v3 V v2 V2 v V v6 a4 a2 v V C2 a6 C2 v2 V2 C v Z*', [qw(len object_type ID charID walk_speed opt1 opt2 option type hair_style weapon shield lowhead tick tophead midhead hair_color clothes_color head_dir costume guildID emblemID manner opt3 stance sex coords xSize ySize lv font maxHP HP isBoss opt4 name)]],
+		'09FE' => ['actor_connected', 'v C a4 a4 v3 V v2 V2 v7 a4 a2 v V C2 a3 C2 v2 V2 C v Z*', [qw(len object_type ID charID walk_speed opt1 opt2 option type hair_style weapon shield lowhead tophead midhead hair_color clothes_color head_dir costume guildID emblemID manner opt3 stance sex coords xSize ySize lv font maxHP HP isBoss opt4 name)]],
+		'09FF' => ['actor_exists', 'v C a4 a4 v3 V v2 V2 v7 a4 a2 v V C2 a3 C3 v2 V2 C v Z*', [qw(len object_type ID charID walk_speed opt1 opt2 option type hair_style weapon shield lowhead tophead midhead hair_color clothes_color head_dir costume guildID emblemID manner opt3 stance sex coords xSize ySize state lv font maxHP HP isBoss opt4 name)]],
+
+		'0A0A' => ['storage_item_added', 'a2 V2 C4 a16 a25', [qw(ID amount nameID type identified broken upgrade cards options)]],
+		'0A0B' => ['cart_item_added', 'a2 V2 C4 a16 a25', [qw(ID amount nameID type identified broken upgrade cards options)]],
+		'0A37' => ['inventory_item_added', 'a2 v V C3 a8 V C2 a4 v a25', [qw(ID amount nameID identified broken upgrade cards type_equip type fail expire unknown options)]],
+
+		
+	);
+
+	foreach my $switch (keys %packets) {
+		$self->{packet_list}{$switch} = $packets{$switch};
+	}
+
+	my %handlers = qw(
+		received_characters 099D
+		received_characters_info 082D
+		sync_received_characters 09A0
+	);
+
+	$self->{packet_lut}{$_} = $handlers{$_} for keys %handlers;
+	$self->{blockingPlayerCancel} = 1;
```

Comment:
move this to the first know client that use it

Reply:
if im not wrong is something like ragexere 2018 04 12

Reply:
fixed a4297ae